### DISCUSSION TITLE
The 1.1.3 version of the panel cannot change the default language

### DISCUSSION THREAD

**Discussion by wuzhaochun** [pterodactyl/panel#2898] [Support]
The 1.1.3 version of the panel cannot change the default language

[zh.zip](https://github.com/pterodactyl/panel/files/5727974/zh.zip)

**Comment by trenutoo** [comment]
There's no translation support in 1.x version at this moment, there are also multiple issues open regarding this. You would have to wait until translation support is implemented or manually edit the files and rebuild the panel.

**Reply by Anarchist-byte** [reply]
But I have saw tons of hosts which somehow translated the panel.

(Sorry for bumping such old discussion)

**Reply by trenutoo** [reply]
That would be a custom modification to the panel as I mentioned above

> or manually edit the files and rebuild the panel.
